### ISSUE TITLE
vscode firebase database suggestion snapshot:any

### ISSUE THREAD

**Issue by Eumhongin** [invertase/react-native-firebase#3823]
Why suggestion not working?

```
import '@react-native-firebase/database';
import firebase from '@react-native-firebase/app';

firebase.database().ref('path').once('value',snapshot => { //snapshot:any...
      console.log(snapshot.val()) // <----- not working
})

```

<img width="453" alt="스크린샷 2020-06-23 오후 12 58 45" src="https://user-images.githubusercontent.com/23710025/85359465-4a322480-b551-11ea-8069-fd777d08e451.png">

package.json

```
"@react-native-firebase/app": "^7.1.4",
"@react-native-firebase/database": "^7.1.2",
```

**Comment by Eumhongin** [comment]
before
<img width="433" alt="스크린샷 2020-06-23 오후 1 33 57" src="https://user-images.githubusercontent.com/23710025/85361187-33420100-b556-11ea-8a50-78bf48b31ff2.png">

after
<img width="463" alt="스크린샷 2020-06-23 오후 1 34 23" src="https://user-images.githubusercontent.com/23710025/85361203-41901d00-b556-11ea-84c2-89f4ccfebda2.png">


working!

**Comment by Salakar** [comment]
Re-opened as this should be fixed, thanks for reporting the issue.

**Comment by dackers86** [comment]
PR added for .once https://github.com/invertase/react-native-firebase/pull/3842

**Comment by dackers86** [comment]
Closed along with merged PR #3842
